### ISSUE TITLE
Can't Create World After Game Restart

### ISSUE THREAD

**Issue by TheVisionaryOne** [ghzdude/MCRandomizer#19]
I've encountered an issue with creating new worlds while random crafting recipes are enabled. On world creation, I receive the following error:

> Internal Exception: io.netty.handler.codec.EncoderException: Failed to encode packet 'clientbound/minecraft:update_recipes'

![javaw_dxFDITmCZr](https://github.com/user-attachments/assets/94a8d49f-5d61-4574-bf37-e471d87b6aca)

I have tested this issue extensively, and it is somewhat difficult to pin down the cause. Though I _have_ successfully reproduced the following steps without encountering this issue, I have not discovered a better way to reproduce it. Any worlds created after Step 2 should encounter this issue.

How to Reproduce This Issue:
1. Create a new world.
2. Restart the game.
3. Create another new world.

I'm running the latest version of MC Randomizer (0.8), as well as the latest version of Forge (51.0.33). I should note that in addition to randomized recipes, randomized recipe inputs are also enabled. Strangely enough, it seems that deleting the .jar file and reinstalling the mod fixes this issue temporarily.

**Comment by ghzdude** [comment]
try 0.8a and see if the issue still occurs

**Comment by TheVisionaryOne** [comment]
I tried to load 0.8a for testing, but it crashes on game start. I've opened it as a separate issue here: https://github.com/ghzdude/MCRandomizer/issues/20

**Comment by ghzdude** [comment]
try again with 0.8b

**Comment by TheVisionaryOne** [comment]
After even more extensive testing on 0.8b, I'm still encountering this issue, and I understand it now.

This issue is caused by creating a new world without generating a new "randomizer" config folder. Manually deleting this config folder before world creation fixes the issue.
